community-operators/cluster-logging/cluster-logging.v4.1.0.clusterserviceversion.yaml

```diff
@@ -3,7 +3,7 @@
 apiVersion: operators.coreos.com/v1alpha1
 kind: ClusterServiceVersion
 metadata:
-  name: clusterlogging.v0.0.1
+  name: clusterlogging.v4.1.0
```

Comment:
By renaming this CSV, OLM has no way of updating existing cluster-logging operator instances installed. If you want to support upgrading from 0.0.1 to 4.1.0, you should create a new CSV with the `spec.replaces` field set to the name of the older CSV.

Reply:
Im not concerned as we have no official 0.0.1 releases.  4.1.0 is technically "the first release"

Reply:
In that case, this approach is fine.